docs/automated-testing/ui-testing/teams-tests.md

```diff
@@ -0,0 +1,235 @@
+
+    If the appium server runs on your local machine at the default portal, then Remote Host and Remote Port can be kept to the default values.
+
+    The configuration should look similar to the printscren below:
+    ![appium-inspector](./images/appium-inspector.png)
+3. Press on **Start Session**.
+    - In the browser, you should see a similar view as below:
+    ![teams-appium-inspector](./images/teams-appium-inspector.png)
+    - You can do any action on the emulator, and if you press on the "Refresh" button in the browser, the left hand side of the Appium Inspector will reflect your app. In the **App Source** you will be able to see the IDs of the elements, so you can write relevant selectors in your tests. 
+
+#### Connecting to Appium server
+
+Below is how this can be accomplished with JavaScript. A similar approach can be followed for other languages.
+Assuming you are using [webdriverio](https://webdriver.io/) as the client, you would need to initialize the remote connection as follows:
+
+```javascript
+ const opts = {
+      port: 4723,
+      hostname: "your-hostname",
+      capabilities: {
+        platformName: "android",
+        "appium:deviceName": "the-name-of-the-virtual-device",
+        "appium:appPackage": "com.microsoft.teams",
+        "appium:appActivity": "com.microsoft.skype.teams.Launcher",
+        "appium:automationName": "the-name-of-the-driver",
+      },
+    };
+
+    // Create a new WebDriverIO instance with the Appium server URL and capabilities
+    await wdio.remote(opts);
+```
+
+1. "port": the port on which the Appium server runs on. By default, it is 4723.
+1. "hostname": the IP of the machine where the Appium sever runs on. If it is running locally, that is 127.0.0.1. If it runs in an Azure VM, it would be the public IP address of the VM. Note: ensure you have followed the steps from **2. Enable connection from outside to Appium server on the VM**.
+1. "platformName": Appium can be used to connect to different platforms (Windows, iOS, Android). In our case, it would be "android".
+1. "appium:deviceName": the name of the Android Virtual Device. See **Useful commands** on how to find the name of the device.
+1. "appium:appPackage": the name of the app's package that you would like to launch. Teams' package name is "com.microsoft.teams".
+1. "appium:appActivity": the activity within Teams that you would like to launch on the device. In our case, we would like just to launch the app. The activity name for launching Teams is called "com.microsoft.skype.teams.Launcher".
+1. "appium:automationName": the name of the driver you are using. Note: Appium can communicate to different platforms. This is achieved by installing a dedicated driver, designed for each platform. In our case, it would be [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver) or [Espresso](https://github.com/appium/appium-espresso-driver), since they are both designed for Android platform. 
```

Comment:
Is this text taken from somewhere or is it self originated?

Reply:
@shirar thanks for reviewing it! It is self originated